app/script/calling/entities/CallEntity.js

```diff
@@ -135,7 +133,12 @@ z.calling.entities.CallEntity = class CallEntity {
       return false;
     });
 
-    this.participantsCount = ko.pureComputed(() => this.getNumberOfParticipants(this.selfUserJoined()));
+    ko.pureComputed(() => {
+      const additionalCount = this.selfClientJoined() ? 1 : 0;
+      return this.participants().length + additionalCount;
+    }).subscribe(numberOfParticipants => {
+      this.telemetry.numberOfParticipantsChanged(numberOfParticipants);
+    });
```

Comment:
if we have the computed any way lets keep track of it with `this.participantsCount`